src/main/java/org/eclipse/microprofile/starter/rest/model/ServerOptions.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.eclipse.microprofile.starter.rest.model;
+
+import org.eclipse.microprofile.starter.addon.microprofile.servers.model.MicroprofileSpec;
+import org.eclipse.microprofile.starter.core.model.JavaSEVersion;
+import org.eclipse.microprofile.starter.core.model.MicroProfileVersion;
+
+import java.util.List;
+
+/**
+ * @author Michal Karm Babacek <karm@redhat.com>
+ */
+public class ServerOptions {
+
+    public final MicroProfileVersion mpVersion;
```

Comment:
You can also use private and a getter. Why the change of coding style? (which required a change to checkstyle)

Reply:
@rdebusscher I wanted to avoid a getter and setter that does nothing and is not necessary for any proxies etc. I can change it, if you would prefer it.

Reply:
But there are already similar parts where getters and setters are created.

So this will introduce several coding styles in one project which is never beneficial for readability. (because people will start to wonder why we did this)